packages/atomic/src/components.d.ts

```diff
@@ -328,10 +330,22 @@ export namespace Components {
           * The result content to display.
          */
         "content": string;
+        /**
+          * How large or small results should be.
```

Comment:
Is this only affecting how much padding is applied to the results?

Reply:
Mostly paddings/margins, but also font size, line height, or other styling that makes results denser.